mongo/datadog_checks/mongo/common.py

```diff
@@ -45,6 +45,9 @@ def get_long_state_name(state):
 
 
 class Deployment(object):
+    def __init__(self):
+        self.use_shards = False
```

Comment:
I'm having a not-so-easy time understanding what this property means.

Is it that the deployment model has shards in it? For example "most deployments don't contain shards", but "mongos deployments always have sharded collections", and "replica set deployments may or may not be in a sharded environment"?

Reply:
Yep that's exactly it. Anything in particular to improve clarity?